### PR TITLE
perf(retry): delete props in retryJob lua script

### PR DESCRIPTION
**Pull request by roggervalf**

### CODE REVIEW

File: src/commands/reprocessJob-4.lua [hunk 1]
```diff
@@ -20,6 +21,7 @@ if (rcall("EXISTS", KEYS[1]) == 1) then
   local jobId = ARGV[1]
   if (rcall("ZREM", KEYS[3], jobId) == 1) then
     rcall(ARGV[2], KEYS[4], jobId)
+    rcall("HDEL", KEYS[1], "finishedOn", "processedOn", ARGV[3])
```
Comment (roggervalf) [inline src/commands/reprocessJob-4.lua:24]: probably we should set attempstMade as 0 too, what do you think @manast ?

Comment (manast) [reply src/commands/reprocessJob-4.lua:24]: Not sure, why would we like to reset the number of attempts here?

Comment (roggervalf) [reply src/commands/reprocessJob-4.lua:24]: if the job failed after the last attempt, if we retry the job, it would keep the last maximum attempt number

Comment (roggervalf) [reply src/commands/reprocessJob-4.lua:24]: as we are not going to reset the attempstMade, this pr should be ready

File: src/classes/job.ts [hunk 2]
```diff
@@ -844,23 +844,16 @@ export class Job<
   /**
    * Attempts to retry the job. Only a job that has failed can be retried.
```
Comment (manast) [inline src/classes/job.ts:845]: Since we can actually retry completed jobs, this comment is incorrect right now.

**Review by manast — APPROVED** [review]

**Pull request merged**